src/solvers/lowering/byte_operators.cpp

```diff
@@ -246,7 +246,7 @@ exprt lower_byte_extract(const byte_extract_exprt &src, const namespacet &ns)
       {
         plus_exprt new_offset(
           unpacked.offset(),
-          from_integer(i * (*element_width), unpacked.offset().type()));
+          from_integer(i * (*element_width) / 8, unpacked.offset().type()));
```

Comment:
Consider using `_bytes` and _bits` suffixes to discourage this sort of problem in future

Reply:
Will do in an additional commit.

Reply:
Done.